mlflow/tracking/fluent.py

```diff
@@ -89,14 +87,8 @@ def start_run(run_id=None, experiment_id=None, source_name=None, source_version=
                           is unspecified, will look for valid experiment in the following order:
                           activated using ``set_experiment``, ``MLFLOW_EXPERIMENT_ID`` env variable,
                           or the default experiment.
-    :param source_name: Name of the source file or URI of the project to be associated with the run.
-                        If none provided defaults to the current file.
-    :param source_version: Optional Git commit hash to associate with the run.
-    :param entry_point_name: Optional name of the entry point for the current run.
-    :param source_type: Integer :py:class:`mlflow.entities.SourceType` describing the type
-                        of the run ("local", "project", etc.). Defaults to
-                        :py:class:`mlflow.entities.SourceType.LOCAL` ("local").
-    :param run_name: Name of new run. Used only when ``run_id`` is unspecified.
+    :param run_name: Name of new run (set a mlflow.runName tag).
```

Comment:
"set a mlflow.runName tag" is a big confusing. maybe "stored as a ``mlflow.runName`` tag" ?